lldb/include/lldb/Utility/ProcessInfo.h

```diff
@@ -254,6 +280,8 @@ class ProcessInstanceInfo : public ProcessInfo {
   struct timespec m_system_time {};
   struct timespec m_cumulative_user_time {};
   struct timespec m_cumulative_system_time {};
+  std::optional<int8_t> m_priority_value = std::nullopt;
+  bool m_zombie = false;
```

Comment:
making this optional as well would be good. If it isn't set, it means we don't know, but if it is set to true/false we do know. Again, almost all members of this structure could benefit from being std::optional

Reply:
done